facades/cmc_facade.py

```diff
@@ -0,0 +1,26 @@
+import requests
+import locale
+import json
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+def get_btc_price():
+  url = 'https://pro-api.coinmarketcap.com/v2/cryptocurrency/quotes/latest'
+
+  parameters = {
+    'symbol': 'BTC'
+  }
+
+  headers = {
+    'Accepts': 'application/json',
+    'X-CMC_PRO_API_KEY': os.getenv('CMC_API_KEY')
```

Comment:
Retrieve API key separately and validate it, raising if it's missing. Don't consume an API call if we don't have the proper credentials.